samples/multi-agent/src/main/java/demo/multiagent/application/SummarizationEvaluator.java

```diff
@@ -0,0 +1,82 @@
+package demo.multiagent.application;
+
+import akka.javasdk.agent.Agent;
+import akka.javasdk.agent.EvaluationResult;
+import akka.javasdk.agent.MemoryProvider;
+import akka.javasdk.annotations.AgentDescription;
+import akka.javasdk.annotations.ComponentId;
+
+import java.util.Locale;
+
+@ComponentId("summarization-evaluator")
+@AgentDescription(
+  name = "Summarization Evaluator Agent",
+  description = """
+  An agent that acts as an LLM judge to evaluate a summarization task.
+  """,
+  role = "evaluator"
+)
+public class SummarizationEvaluator extends Agent {
```

Comment:
I don't think there is a need for a separate abstraction or separate component for making it easier to write evaluator agents, but we could consider to have some built-in evaluator agents like this SummarizationEvaluator and the ToxicityEvaluator. They would be included in the SDK but called from consumers or workflows like any other agent. The prompts should be editable, i.e. perfect for the PromptTemplate entity.